test/project_types/script/layers/infrastructure/languages/typescript_task_runner_test.rb

```diff
@@ -18,6 +26,15 @@
     }
   end
 
+  def stub_tool_versions(npm:, node:)
+    ctx.stubs(:capture2e)
+      .with("npm", "--version")
+      .returns([npm, mock(success?: true)])
+    ctx.stubs(:capture2e)
+      .with("node", "--version")
+      .returns([node, mock(success?: true)])
+  end
+
```

Comment:
We shouldn't be subbing `context` here.  The dependency here is actually `TaskRunner::check_tool_version!`, and that's the method that should be stubbed.